goenv/goenv.go

```diff
@@ -57,6 +58,13 @@ func Get(name string) string {
 			panic("could not find cache dir: " + err.Error())
 		}
 		return filepath.Join(dir, "tinygo")
+	case "CGO_ENABLED":
+		val := os.Getenv("CGO_ENABLED")
+		if val == "1" || val == "0" {
+			return val
+		}
+		// Default to enabling CGo.
```

Comment:
This addressed my only possible concern about this PR.

Not sure how useful it will be, but it does enforce the standard behavior people would expect.

Reply:
Not sure what you mean, is there something that needs changing?

This behavior here is slightly different from the main Go compiler. In TinyGo, CGo is always enabled unless it is explicitly disabled. In the main Go compiler, CGo is disabled by default when cross compiling. That doesn't make much sense for TinyGo as it supports CGo while cross compiling and cross compiling is in fact the main use case of TinyGo.

Reply:
> This behavior here is slightly different from the main Go compiler.

Exactly, and we really do have to keep it that way. That was what I was saying, and you already handled it the way I would want to. So we're good. :)